src/youtube/js/youtube.js

```diff
@@ -119,15 +119,19 @@ require("../../../html5-common/js/utils/constants.js");
     playerReady = true;
     for(var i = 0; i < javascriptCommandQueue.length; i++) 
     {
-      if(javascriptCommandQueue[i][0] === "play")
-      { 
-        element.play();
-        hasPlayed = true;
+      switch(javascriptCommandQueue[i][0])
```

Comment:
The `javascriptCommandQueue[i][0]` will throw an error if `javascriptCommandQueue` contains bad values like `null` or `undefined`. But it does not seem like 'javascriptCommandQueue` can ever be populated with those type of values, so these checks will suffice.